requirements/base.txt

```diff
@@ -1,13 +1,13 @@
 boto==2.42.0                        # MIT
-Django==1.10.7 						# BSD License
-django-countries==4.0               # MIT
-djangorestframework==3.5.3			# BSD
-django-rest-swagger==0.3.8  		# BSD
-djangorestframework-csv==1.4.1		# BSD
-django-storages==1.5.1              # BSD
+Django==1.11 						# BSD License
+django-countries==4.5               # MIT
+djangorestframework==3.6.2			# BSD
+django-rest-swagger==2.1.2  		# BSD
+djangorestframework-csv==2.0.0		# BSD
+django-storages==1.5.2              # BSD
 elasticsearch-dsl==0.0.11           # Apache 2.0
-ordered-set==2.0.1                  # MIT
-tqdm==4.10.0                        # MIT
+ordered-set==2.0.2                  # MIT
+tqdm==4.11.2                        # MIT
```

Comment:
Great idea!  I've updated the packages that I could in `text.txt` too.  FYI, updating pylint to 1.7.1 causes a whole slew of errors, so I only went up to 1.6.5.